Files/Views/Pages/Properties.xaml.cs

```diff
@@ -119,7 +129,15 @@ private async void Properties_Loaded(object sender, RoutedEventArgs e)
                 ItemProperties.ItemMD5HashProgressVisibility = Visibility.Collapsed;
             }
         }
-
+        public  string ToBytesCount(long bytes)
```

Comment:
You could use `ByteSizeLib.ByteSize.FromBytes(long).ToString();`

Reply:
Fixed in this [commit](https://github.com/files-community/files-uwp/commit/59fc0f8c336ca5a12a505f43f2071d2e9272990d)